.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: java
+
+jdk:
+  - oraclejdk8
+
+os:
+  - windows
+  - osx
+
+script:
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then mvn -Pmac -Dmaven.test.skip=true clean compile;
```

Comment:
Can we put a TODO here to remove the skip test?  You can assign this to me -- as I'm trying to fix the tests.

Is there a way to run the tests in the native code as well?